features/step_definitions/et3_claiments_details_page_steps.rb

```diff
@@ -33,7 +33,7 @@
   claimants_details_page.agree_with_employment_dates_question.set(:no)
 end
 
-Then(/^I should see the error message saying the further information about dates cant be blank$/) do
-  claimants_details_page.employment_start.assert_error_message(t('errors.claimants_details.start_date_blank'))
-  claimants_details_page.employment_end.assert_error_message(t('errors.claimants_details.end_date_blank'))
-end
+# Then(/^I should see the error message saying the further information about dates cant be blank$/) do
```

Comment:
Why is this commented out ?  If you don't need it just remove it

Reply:
Silly mistake, I forgot to remove it - thanks